.github/workflows/tests.yml

```diff
@@ -134,6 +134,8 @@ jobs:
             runs-on: "windows-2022"
           - ruby: "3.2"
             runs-on: "windows-2022"
+    env:
```

Comment:
Minor thing but could this also be expressed in the matrix above as a new variable, e.g. `requires-pkgcon-upgrade: true` on the various `windows` entries and then you can simplify the `if` in the job?

Reply:
Let's try that.